src/components/graph-view.js

```diff
@@ -1192,10 +1192,28 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
 
     const next = this.computeZoom(viewBBox);
 
+    /* 
+    If the initial position should be aligned with the entities rather than the graph, need to account for where the entities are positioned
+    Subtract the bounding box coordinates and adjust for the scale of the graph
+    */
+    if (initialPosition.x) {
+      next.x =
+        initialPosition.alignWith === 'entities'
+          ? (initialPosition.x - viewBBox.x) * next.k
+          : initialPosition.x;
```

Comment:
why do we have this discrepancy here? doesn't `zoomAndTranslate` already handle `k`?

Reply:
Good question, so it does handle that part for us. The problem is with the initial coordinates we pass in - they're essentially relative to the screen rather than to the SVG, so we need to take into account the scale and update the passed in coordinates accordingly.

Lmk if that doesn't make sense!

Reply:
Yeah that's what I thought as well. My only worry is that the behavior might be unexpected ie should we also `/ k` in case when `alignWith` is not  == `entities` ?

Reply:
Ahh I see, yeah that's a good point. I think that would probably make sense -- for plan the zoom level should be 1, so that should be fine/since I added an initial zoom level attribute, it could easily be overriden. I'll add that!